applications/mne_scan/plugins/ftbuffer/FormFiles/ftbuffersetupwidget.cpp

```diff
@@ -0,0 +1,110 @@
+//=============================================================================================================
```

Comment:
I just compiled your branch on win and mac and came across the `FtBufferSetupWidget`. Do we need this widget? If not please remove the widget and its action. The same applies for `FtBufferAboutWidget`.